# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arphound/hotfix-include.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- arphound-1.3.2.orig/shared/dnsquery.C	2006-06-03 01:26:01.000000000 +0200
+++ arphound-1.3.2/shared/dnsquery.C	2026-07-26 06:34:49.788016580 +0200
@@ -21,7 +21,7 @@
  */
 
 #include <stdio.h>
-#include <alloc.h>
+#include <malloc.h>
 #include <time.h>
 #include <unistd.h>
 #include <string.h>
